libraries/AP_Proximity/AP_Proximity.h

```diff
@@ -88,6 +88,9 @@ class AP_Proximity
     // get distances in PROXIMITY_MAX_DIRECTION directions. used for sending distances to ground station
     bool get_horizontal_distances(Proximity_Distance_Array &prx_dist_array) const;
 
+    // get raw and filtered distances in 8 directions per layer. used for logging
+    bool get_active_layer_distances(uint8_t layer, AP_Proximity::Proximity_Distance_Array &prx_dist_array, AP_Proximity::Proximity_Distance_Array &prx_filt_dist_array) const;
```

Comment:
Not sure what "active" means here....

Reply:
It means that it has a valid distance on the layer